docs/cedarling/cedarling-quick-start.md

```diff
@@ -10,14 +10,187 @@ tags:
 
 # Cedarling Quick Start Guide
 
-## This content is in progress
+## Introduction
 
-The Janssen Project documentation is currently in development. Topic pages are being created in order of broadest relevance, and this page is coming in the near future.
+In this guide, we'll demonstrate how to use Cedarling as a sidecar for an API Gateway to authorize requests. 
+This setup involves configuring the Cedarling to protect resources based on enterprise policies.
 
-## Have questions in the meantime?
+### Sequence diagram
 
-While this documentation is in progress, you can ask questions through [GitHub Discussions](https://github.com/JanssenProject/jans/discussions) or the [community chat on Gitter](https://gitter.im/JanssenProject/Lobby). Any questions you have will help determine what information our documentation should cover.
+```mermaid
+sequenceDiagram
+title Cedarling and Tarp
 
-## Want to contribute?
+participant Browser
+participant Tarp
+participant Auth Server
 
-If you have content you'd like to contribute to this page in the meantime, you can get started with our [Contribution guide](https://docs.jans.io/head/CONTRIBUTING/).
+Browser->Tarp: Enter Auth Server config
+Tarp->Auth Server: DCR Request
+Auth Server->Tarp: Client ID and Secret
+Browser->Tarp: Enter Cedarling bootstrap config
+Tarp->Auth Server: GET /jans-auth/restv1/jwks
+Auth Server->Tarp: JWKS
+Tarp->Tarp: Initialize Cedarling
+Browser->Tarp: Start authorization flow
+Tarp->Auth Server: GET /jans-auth/restv1/authorize?...
+Auth Server->Tarp: /callback?code=...
+Tarp->Auth Server: GET /jans-auth/restv1/token
+Auth Server->Tarp: {"access_token": <JWT>}
+
+Browser->Tarp: Cedarling Authz(principal, action, resource, context)
+Tarp->Browser: Cedarling Decision
+```
+
+## Prerequisites
+
+Before you begin, make sure you have the following:
+
+* [Firefox](https://www.mozilla.org/en-US/firefox/windows/) or [Google Chrome](https://www.google.com/chrome/index.html)
+* The latest release of [Jans Tarp](https://github.com/JanssenProject/jans/releases/tag/nightly). Download the zip file for your browser.
```

Comment:
What would be best here is to provide a link: "How to install Jans Tarp (1 minute video)". @ossdhaval, do we have an updated video?